dataprep/eda/missing/render.py

```diff
@@ -39,14 +37,14 @@
 
 def render_missing(
     itmdt: Intermediate, plot_width: int = 500, plot_height: int = 500,
-) -> LayoutDOM:
+) -> Dict[str, Any]:
     """
     @Jinglin write here
     """
     if itmdt.visual_type == "missing_impact":
         return render_missing_impact(itmdt, plot_width, plot_height)
     elif itmdt.visual_type == "missing_impact_1vn":
-        return render_missing_impact_1vn(itmdt, plot_width, plot_height)
+        return render_missing_impact_1vn(itmdt, plot_width - 100, plot_height - 150)
```

Comment:
I don't love that the rightmost plot in each row is cut off
![Screen Shot 2020-10-04 at 11 12 03 PM](https://user-images.githubusercontent.com/25874021/95045656-52911400-0697-11eb-9be4-4f15852968e3.png)
I think we could default `bins` to 20 rather than 30 and change this line to 
``` python
return render_missing_impact_1vn(itmdt, plot_width - 176, plot_height - 200)
```
so the plot size matches `plot(df)`. This is just my opinion; if @dovahcrow and @eutialia like it then I'm fine with how it currently is. A problem is that the `bins` value is used in many different plots, but this will not be the case when parameter management is done so we could address this later. Otherwise it looks good to me.

Reply:
Good point! Is the jupyter width here the same for most of the screen?

Reply:
Yes it's same if viewport is wider than 1200px